rpcs3/Emu/RSX/GL/GLTexture.h

```diff
@@ -31,6 +31,34 @@ namespace gl
 		u64 memory_required;
 	};
 
+	struct clear_cmd_info
+	{
+		GLbitfield aspect_mask = 0;
+
+		struct
+		{
+			f32 value = 0;
+		}
+		clear_depth;
```

Comment:
```suggestion
		clear_depth{};
```

Reply:
The members are already initialized.

Reply:
Changed to forced zero-init.

Reply:
better.
just for reference, your old code warns us:
warning: ‘clear_cmd_info::clear_stencil’ should be initialized in the member initialization list [-Weffc++]